syntheseus/tests/reaction_prediction/cli/test_eval.py

```diff
@@ -98,11 +153,10 @@ def test_print_and_save():
         predictions=[[SingleProductReaction(product=input_mol, reactants=output_mol_bag)]],
     )
 
-    with tempfile.TemporaryDirectory() as temp_dir:
-        config = EvalConfig(
-            data_dir=temp_dir,
-            model_class=BackwardModelClass.RetroKNN,
-            results_dir=temp_dir,
-        )
+    config = EvalConfig(
+        data_dir=str(tmp_path),
+        model_class=BackwardModelClass.RetroKNN,
```

Comment:
Why RetroKNN here? Is this just an arbitrary choice? (I know this is not a new change but I didn't notice this before)

Reply:
Yes, completely arbitrary, doesn't matter.

Reply:
Maybe add a comment clarifying this in case we forget in the future?